cypress/integration/milmove/ppms/advances.js

```diff
@@ -0,0 +1,106 @@
+import {
+  navigateFromAdvancesPageToReviewPage,
+  navigateFromDateAndLocationPageToEstimatedWeightsPage,
+  navigateFromEstimatedIncentivePageToAdvancesPage,
+  navigateFromEstimatedWeightsPageToEstimatedIncentivePage,
+  setMobileViewport,
+  signInAndNavigateFromHomePageToExistingPPMDateAndLocationPage,
+  submitsAdvancePage,
+  submitsEstimatedWeights,
+  submitsEstimatedWeightsAndProGear,
```

Comment:
remove unused imports
```suggestion
  navigateFromDateAndLocationPageToEstimatedWeightsPage,
  navigateFromEstimatedIncentivePageToAdvancesPage,
  navigateFromEstimatedWeightsPageToEstimatedIncentivePage,
  setMobileViewport,
  signInAndNavigateFromHomePageToExistingPPMDateAndLocationPage,
  submitsAdvancePage,
```